modules/microshift-adding-repos-to-image-builder.adoc

```diff
@@ -11,15 +11,69 @@ Use the following procedure to add the {product-title} repositories to Image Bui
 .Prerequisites
 * Your build host meets the Image Builder system requirements.
 * You have installed and set up Image Builder and the `composer-cli` tool.
-* You have root user access to your build host.
+* You have root-user access to your build host.
 
 .Procedure
 
-* Enable the {product-title} RPM repositories on the build host by running the following command:
+. Enable the {product-title} RPM repositories on the build host by running the following command:
```

Comment:
This is a wrong comment. Please, check my original script comment for a better description